tools/torchci/td/get_merge_base_info.py

```diff
@@ -87,24 +82,29 @@ def upload_merge_base_info(shas: List[str]) -> None:
                 f"git show --no-patch --format=%ct {merge_base}"
             )
             timestamp = datetime.utcfromtimestamp(int(unix_timestamp)).isoformat() + "Z"
-            docs.append(
-                {
-                    "sha": sha,
-                    "merge_base": merge_base,
-                    "changed_files": changed_files.splitlines(),
-                    "merge_base_commit_date": timestamp,
-                    "repo": "pytorch/pytorch",
-                }
+            data = {
+                "sha": sha,
+                "merge_base": merge_base,
+                "changed_files": changed_files.splitlines(),
+                "merge_base_commit_date": timestamp,
+                "repo": "pytorch/pytorch",
```

Comment:
An unrelated BE request, I think there should be a way to get the repo name here, i.e. `git config --get remote.origin.url`, so that we don't need to hardcode it to `pytorch/pytorch` in the data record or in the merge_bases path

Reply:
This script is only supposed to run for pytorch, so I think it's ok for now.  The run_command function looks specifically for pytorch